src/steps/user/index.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  createDirectRelationship,
+  Entity,
+  IntegrationStep,
+  IntegrationStepExecutionContext,
+  RelationshipClass,
+} from '@jupiterone/integration-sdk-core';
+
+import { createAPIClient } from '../../client';
+import { IntegrationConfig } from '../../config';
+import {
+  ACCOUNT_ENTITY_KEY,
+  Entities,
+  Relationships,
+  Steps,
+} from '../constants';
+import { createUserEntity } from './converter';
+
+export async function fetchUser({
+  instance,
+  jobState,
+}: IntegrationStepExecutionContext<IntegrationConfig>) {
+  const apiClient = createAPIClient(instance.config);
+  const user = await apiClient.getUser();
+  const userEntity = createUserEntity(user);
+
+  await jobState.addEntity(userEntity);
+
+  const userGroups = await apiClient.getUserGroups();
+
+  for (let i = 0; i < userGroups.length; i++) {
```

Comment:
Can we use for (const userGroup of userGroups) instead?

Reply:
Changed the way the iteration over userGroups happen. Instead used the ```iterateEntities``` function instead of the for loop (also reduces API calls). 
```suggestion
    await jobState.iterateEntities(
    { _type: Entities.USER_GROUP._type },
    async (userGroupEntity) => {
      if (userGroupEntity.name === accountEntity.name) {
        await jobState.addRelationship(
          createDirectRelationship({
            _class: RelationshipClass.HAS,
            from: accountEntity,
            to: userGroupEntity,
          }),
        );
      }
    },
  );
```